Directory.Build.props

```diff
@@ -107,6 +107,7 @@
     <IsTestAssetProject Condition="$(RepoRelativeProjectDir.Contains('testassets'))">true</IsTestAssetProject>
     <IsSampleProject Condition="$(RepoRelativeProjectDir.Contains('sample'))">true</IsSampleProject>
     <DisableDeterministicSourceRoot Condition="'$(IsSampleProject)' == 'true' OR '$(IsTestAssetProject)' == 'true'">true</DisableDeterministicSourceRoot>
+    <BuildHelixPayload Condition="'$(IsTestProject)' == 'true'">true</BuildHelixPayload>
```

Comment:
This is going to be evaluated before the contents of the project. I would recommend moving this to Directory.Build.targets and change to.
```xml
<BuildHelixPayload Condition="'$(BuildHelixPayload)' == '' AND '$(IsTestProject)' == 'true'">true</BuildHelixPayload> 
```

Reply:
Cool, I figured it was something like this that was preventing the IsTestProject flag from defaulting BuildHelixPayload correcly